pkgs/applications/misc/hyprland-autoname-workspaces/default.nix

```
@@ -0,0 +1,26 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "hyprland-autoname-workspaces";
+  version = "1.1.7";
+
+  src = fetchFromGitHub {
+    owner = "hyprland-community";
+    repo = pname;
```

```suggestion
    repo = "hyprland-autoname-workspaces";
```

Most people prefer to use a string instead of the pname variable since repo name and pname arent really related but it's not a blocker just a suggestion.

---

Fixed